Library/Homebrew/rubocops/bottle.rb

```diff
@@ -10,8 +10,9 @@ module FormulaAudit
       class BottleFormat < FormulaCop
         extend AutoCorrector
 
-        def audit_formula(_node, _class_node, _parent_class_node, body_node)
-          bottle_node = find_block(body_node, :bottle)
+        sig { override.params(args: FormulaCopArgs).void }
+        def audit_formula(args)
+          bottle_node = find_block(args.body_node, :bottle)
```

Comment:
```suggestion
        sig { override.params(formula_cop_args: FormulaCopArgs).void }
        def audit_formula(formula_cop_args)
          bottle_node = find_block(formula_cop_args, :bottle)
```

like @Bo98 I'm not huge on `args` as a name here but happy to merge this with `formula_cop_args` or similarly descriptive variable name.

Reply:
> I think the question is whether `#audit_formula` is likely to ever have non-node arguments

This is private API. We can name things differently later easily.

The answer of a theoretical future addition would totally depend on what's being added.

Reply:
I'm fine with any name longer than `args`.